README.txt

```diff
@@ -1,7 +1,7 @@
 === Events Manager Pro SecureSubmit Gateway ===
 Contributors: markhagan
 Tags: events, event, event registration, event calendar, events calendar, event management, addon, extension, addition, registration, ticket, tickets, ticketing, tickets, theme, widget, locations, maps, booking, attendance, attendee, calendar, gigs, payment, payments
-Tested up to: 3.9.1
+Tested up to: 4.7
```

Comment:
@SenthilKumar-86 Any chance we can put the latest version of PHP you tested with?